### ISSUE TITLE
Define a folder structure

### ISSUE THREAD

**Issue by ijmitch** [openmainframeproject/wg-modernization#9]
The repository needs to be navigable and consumable by readers.

The need here is to define a set of principles for the hierarchy of folders to clearly segment the content.

Whilst the majority of the content might be Markdown, we will accommodate other asset formats as appropriate.

**Comment by ijmitch** [comment]
Of course, it's possible to define an initial set of folders in a branch/PR for discussion.

**Comment by MistyDecker** [comment]
Some ideas to start...

**Application Modernization:**
•	Code analysis for planning
•	Refactor monolithic applications 
•	Unlock application value through APIs
•	integrate with other modern languages 
•	Leverage managed services (XaaS) such as databases, analytics and storage

**Process Modernization:**
•	Move to modern source code management systems
•	Implement Agile practices
•	Implement DevSecOps
•	Adopt modern development tools
•	Automate unit testing
•	Implement DevOps
•	Adopt CI/CD

**Infrastructure Modernization:**
•	Target containers and orchestration for reliable and faster application delivery
•	Re-platform applications to commodity x86 hardware
•	Use containerization and Infrastructure as Code
•	Implement a multi-cloud model
•	Migrate COBOL and mainframe applications to the cloud
•	Use virtualized and containerized platforms

**Comment by ijmitch** [comment]
I was really intending this issue to solely consider the shape of the folder structure to accommodate topics, rather than define the content/topics to be accommodated.

But I guess we could have a go at doing both and see how it goes.

**Comment by rpomp** [comment]
agree with @pwmccaffrey and @ijmitch above assumes scope agreement, lets focus on work/process content structure.  so @ijmitch are you looking for content category bill of materials?  ie, guidebook, roadmap, code snippets, architecture, best practices, blah blah?

**Comment by gmackinnon** [comment]
agree with @rpomp et al.

**Comment by MistyDecker** [comment]
There are a number of things on @rpomp’s list that hadn’t occurred to me.
How about we start by collecting some examples of what people think should be in the repository?
I’ll start a Wiki just to collect some ideas and then we can draft an INITIAL structure out of that.
Remember that this is our first take on a structure.  Nothing says we can’t decide two months from now or a year from now that another structure will work better.
